extension/src/popup/styles/global.scss

```diff
@@ -15,9 +15,14 @@
 
   --back--button-dimension: 1rem;
   --back--button-z-index: 2;
+
+  // TODO: migrate to SDS
+  --background-color: #1f212a;
+  --button-bg--tertiary: #303448;
```

Comment:
talked with Charles. Looks like the existing SDS colors weren't quite working. We'll override for now